src/Database/Type/DateTimeType.php

```diff
@@ -144,33 +151,41 @@ public function toDatabase($value, DriverInterface $driver): ?string
         return $value->format($this->_format);
     }
 
+    /**
+     * Alias for `setDatabaseTimezone()`.
+     *
+     * @param string|\DateTimeZone|null $timezone Database timezone.
+     * @return $this
+     */
+    public function setTimezone($timezone)
+    {
+        return $this->setDatabaseTimezone($timezone);
+    }
+
     /**
      * Set database timezone.
      *
-     * Specified timezone will be set for DateTime objects before generating
-     * datetime string for saving to database. If `null` no timezone conversion
-     * will be done.
+     * This is the time zone used when converting database strings to DateTime
+     * instances and converting DateTime instances to database strings.
+     *
+     * @see DateTimeType::setKeepDatabaseTimezone
      *
      * @param string|\DateTimeZone|null $timezone Database timezone.
      * @return $this
      */
-    public function setTimezone($timezone)
+    public function setDatabaseTimezone($timezone)
```

Comment:
Why are we adding an alias now?

Reply:
I want to clarify what this setting does.  It is the time zone used to interpret db values only.  In toPHP() there is logic that converts the time zone.  This does not affect that. It also does not affect marshaling.